### ISSUE TITLE
Course not started emails not following internal checks.

### ISSUE THREAD

**Issue by turf212** [iomad/iomad#1208]
As detailed from #1204 

Another issue which I have seen is this report email does not follow rules set within the email template for when to resend email. This email gets generated every time the email_reports_cron runs which is every day at midnight (as default in Iomad task settings). I have not looked into fixing this our side yet but just disable the email template for the company and when we want to send reminder emails to users we enable it and it will run on the next cron interval (disabling it again next day - also had no time to test code fix for this).

**Comment by turf212** [comment]
There was some work done on this but not completely tested / signed off.  Waiting on this and will push the fixes.
